game-app/game-core/src/main/resources/i18n/games/strategy/engine/framework/ui.properties

```
@@ -0,0 +1,23 @@
+startup.SetupPanelModel.button.ConnectToNetworkedGame.Label=Connect to Networked Game
+startup.SetupPanelModel.button.EnginePreferences.Label=Engine Preferences
+startup.SetupPanelModel.button.EnginePreferences.Tooltip=Configure certain options related to the engine.
+startup.SetupPanelModel.button.HostNetworkGame.Label=Host Networked Game
+startup.SetupPanelModel.button.MapCreatorTools.Label=Map Creator Tools
+startup.SetupPanelModel.button.PlayByEmail.Label=Play By Email
+startup.SetupPanelModel.button.PlayByEmail.Tooltip=Starts a game which will be emailed back and forth between all players.
+startup.SetupPanelModel.button.PlayByForum.Label=Play By Forum
+startup.SetupPanelModel.button.PlayByForum.Tooltip=Starts a game which will be posted to an online forum or a message board.
+startup.SetupPanelModel.button.PlayOnline.ConnectToNetworkedGame.Tooltip.Line1=Connects to someone's hosted game,
```

```suggestion
startup.SetupPanelModel.button.PlayOnline.ConnectToNetworkedGame.Tooltip.Line1=Connects to someone''s hosted game,
```
Single quotes need to be escape with another single quote

---

Hm, it works on my computer (Windows 10 with IntelliJ) and should only lead to a proplem

> Whenever you are using MessageFormat you should be aware that the single quote character (') fulfils a special purpose inside message patterns. The single quote is used to represent a section within the message pattern that will not be formatted. A single quote itself must be escaped by using two single quotes ('').

([source link](https://www.mscharhag.com/java/resource-bundle-single-quote-escaping))

Anyhow, I adjusted it.